feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/flow/AssetFlowFragment.kt

```diff
@@ -95,13 +95,15 @@ abstract class AssetFlowFragment<T : AssetFlowViewModel> :
         }
     }
 
-    override fun onDestroyView() {
-        super.onDestroyView()
+    override fun onBackPressed(action: () -> Unit) {
+        super.onBackPressed(action)
```

Comment:
This is a wrong override I believe. `onBackPressed(action: () -> Unit)` is a method in BaseFragment to register a back-pressed listener. It is not a callback itself